package.json

```diff
@@ -1,11 +1,14 @@
 {
-    "name": "standard-assets",
+    "name": "standard-assets-bundle",
```

Comment:
Not sure about this

Reply:
What motivates this, it doesn't appear in other assets?

Reply:
My editor gets mad at me for having two package.json's with the same name

Reply:
Only the inner one really matters for the asset name I think, right?

I guess this doesn't matter, but it does imply you'll be making the same change any time you touch the remaining packages that don't have this.

Reply:
I don't the package.json name in a asset bundle matters anywhere at the moment, but I thought it made since to keep the asset.json name and the asset/package.json name in sync. I will probably be making the same change if I find it anywhere else and my editor gets mad, haha. But I won't go looking for them.